### PR TITLE
Version 1.9.1

### PR DESCRIPTION
**Pull request by AsparagusEduardo**
## Description
Changelog in https://github.com/AsparagusEduardo/pokeemerald/blob/f548f27ed864b0f19066c8604449bd1e6463240c/docs/changelogs/1.9.x/1.9.1.md

## **Discord contact info**
AsparagusEduardo

### CODE REVIEW

File: docs/changelogs/1.9.x/1.9.1.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    * Shiny Galarian Yamask, Darumaka
+* Fixed Unown Overworld follower sprites by Sarn by @hedara90 in [#5146](https://github.com/rh-hideout/pokeemerald-expansion/pull/5146)
+
+## ⚔️ Battle General ⚔️
+### Changed
+* Set new animation particles by default to off by @AlexOn1ine in [#5161](https://github.com/rh-hideout/pokeemerald-expansion/pull/5161)
+### Fixed
+* Fixed speed ties by @mrgriffin in [#4780](https://github.com/rh-hideout/pokeemerald-expansion/pull/4780)
+  * Cleanup by @hedara90 in [#5092](https://github.com/rh-hideout/pokeemerald-expansion/pull/5092)
+* Fixed Defiant/Competitive not working if the battler enters the field with a Court Changed Sticky Web on its side of the field by @PhallenTree in [#5093](https://github.com/rh-hideout/pokeemerald-expansion/pull/5093)
```
Comment (hedara90) [inline docs/changelogs/1.9.x/1.9.1.md:50]:
```suggestion
* Fixed Defiant/Competitive not working after the battler enters the field with a Court Changed Sticky Web on its side of the field by @PhallenTree in [#5093](https://github.com/rh-hideout/pokeemerald-expansion/pull/5093)
```
Might be a better explanation of the problem, Defiant/Competetive correctly didn't trigger, but further stat drops also (incorrectly) didn't trigger the ability.

File: docs/changelogs/1.9.x/1.9.1.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+### Fixed
+* Fixed non-grass Ivy Cudgel breaking battle UI by @hedara90 in [#5117](https://github.com/rh-hideout/pokeemerald-expansion/pull/5117)
+* Fixes Stomping Tantrum effect not doubling power in certain situations by @AlexOn1ine in [#5140](https://github.com/rh-hideout/pokeemerald-expansion/pull/5140)
+* Fixed Fickle Beam's description by @PhallenTree in [#5093](https://github.com/rh-hideout/pokeemerald-expansion/pull/5093)
+* Fixed Revelation Dance interactions with Z-Move, Roost and typeless mons by @PhallenTree in [#5133](https://github.com/rh-hideout/pokeemerald-expansion/pull/5133)
+* Fixes Poltergeist missing its accuracy check by @AlexOn1ine in [#5168](https://github.com/rh-hideout/pokeemerald-expansion/pull/5168)
+* Fixed Fickle Beam not showing its message by @TheTrueSadfish in [#5159](https://github.com/rh-hideout/pokeemerald-expansion/pull/5159)
+* Fixed Retaliate not working correctly if the party member fainted via passive damage by @hedara90 in [#5182](https://github.com/rh-hideout/pokeemerald-expansion/pull/5182)
```
Comment (hedara90) [inline docs/changelogs/1.9.x/1.9.1.md:71]:
```suggestion
* Fixed Retaliate not working correctly if the party member fainted via passive damage during end of turn by @hedara90 in [#5182](https://github.com/rh-hideout/pokeemerald-expansion/pull/5182)
```
It was only end of turn passive damage that broke Retaliate, if the passive damage happened as part of a move (Flame Burst, Recoil etc.) it still worked.

**Review by hedara90 — COMMENTED** [review]
Some clarifications

**Comment by AsparagusEduardo** [comment]
Applied changes.

**Pull request merged**